docs-generator/src/404.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+
+  <title>404 Redirect</title>
+
+  <script>
+    sessionStorage.redirect = location.href;
+  </script>
+
+  <meta http-equiv="refresh" content="0;URL='/tauri'"></meta>
```

Comment:
Looks like this wants to go to `/tauri` but should probably go to the root `/` instead.

Reply:
ahhh - missed that one.
nice catch